frontend/packages/dev-console/src/components/pipelines/__tests__/Pipeline.spec.tsx

```diff
@@ -18,11 +19,29 @@ const pipelinePageProps = {
   },
 };
 
-const pipelineWrapper = shallow(<PipelinesPage {...pipelinePageProps} />);
+const pipelinePagePropsWNS = {
+  history: null,
+  location: null,
+  match: {
+    isExact: true,
+    path: `/k8s/ns/:ns/${referenceForModel(PipelineModel)}`,
+    url: 'k8s/ns/my-project/tekton.dev~v1alpha1~Pipeline',
+    params: {
+      ns: 'all-projects',
+    },
+  },
+};
 
 describe('Pipeline List', () => {
+  const pipelineWrapperNS = shallow(<PipelinesPage {...pipelinePageProps} />);
```

Comment:
this should be inside the test i.e. `it` block

Reply:
You should change to render as the following once you fix the prop types:
`shallow(<PipelinesPage match={{ params: { ns: 'test-namespace' }}} />);

Reply:
unable to change props type in PipelinesPage

@sahil143 moved inside it block